electron/js/lib/settings.js

```diff
@@ -21,15 +21,27 @@
 
 const fs = require('fs');
 const path = require('path');
-
-const app = require('electron').app || require('electron').remote.app;
 const debug = require('debug');
+const mkdirp = require('mkdirp');
+const app = require('electron').app || require('electron').remote.app;
 
-const INIT_JSON = path.join(app.getPath('userData'), 'init.json');
+const configDir = path.join(app.getPath('userData'), 'config');
+mkdirp.sync(configDir);
```

Comment:
Instead of a combination of `fs` and `mkdirp`, we can simply use `fs-extra` for both tasks (creating files and creating directories).

Reply:
But `mkdirp` was already in the dependencies.